apis/betterself/v1/events/tests.py

```diff
@@ -96,3 +114,30 @@ def test_uniqueness_on_post_requests(self):
 
         self.assertTrue(amount_of_events_post_request > amount_of_events)
         self.assertEqual(amount_of_events_post_request_repeated, amount_of_events_post_request)
+
+
+class TestProductivityLogViews(BaseAPIv1Tests, GetRequestsTestsMixin, PostRequestsTestsMixin):
+    # python manage.py test apis.betterself.v1.events.tests.TestProductivityLogViews
+    TEST_MODEL = DailyProductivityLog
+
+    def setUp(self):
```

Comment:
This is missing a bunch of tests that probably should be added. IE. No edge cases were done.